src/generateRenderer/colorRamps.js

```diff
@@ -0,0 +1,54 @@
+const chroma = require('chroma-js')
+
+module.exports = { createMultipartRamp, createAlgorithmicRamp }
+
+/**
+*
+* generate multipart color ramp
+*
+* @param {array} options
+* @return {array} algorithmic colorRamps
+*/
+function createMultipartRamp (options) {
+  const { rampDetails, breakCount = 7 } = options
```

Comment:
yeah, it's very handy although I am probably not using it to its full potential. My only react courses stressed this.